eparticlectr/eparticlectr.cpp

```diff
@@ -357,76 +255,27 @@ void eparticlectr::finalizeextr( uint64_t proposal_id ) {
                 });
             }
         }
-        // Reduce staking time for vote winners to 5 days
-        else if (approved && !vote_it->is_editor) {
```

Comment:
why remove the approved checking here? its used in the first if but not in this one